src/data.cpp

```diff
@@ -1056,6 +1056,10 @@ namespace OpenBabel
     // Check return value from OpenDatafile
     // Suggestion from Zhiguo Liu
     string fn_open = OpenDatafile(ifs, _filename, _envvar);
+    
+    // Check _subdir directory
```

Comment:
I dunno. I'm inclined to merge this for 2.4.0 - which has been postponed *far* too long and then work hard on a 2.4.1. Sound OK?